bin/run-test.sh

```diff
@@ -42,7 +42,10 @@ function run {
 run test test/hooks
 for test in test/standalone/test-*.js ;
 do
-  run "${test}"
+  if [[ ! $(node --version) =~ v0\.12\..* || ! "${test}" =~ .*trace\-koa\.js ]]
+  then
+    run "${test}"
+  fi
```

Comment:
Yea, it is a possibility but pushes the line over 90 characters forcing it to be broken up anyways.